WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift

```diff
@@ -238,22 +240,32 @@ private extension SignupEpilogueViewController {
 
         let context = ContextManager.sharedInstance().mainContext
 
-        guard let defaultAccount = AccountService(managedObjectContext: context).defaultWordPressComAccount(),
-            let restApi = defaultAccount.wordPressComRestApi else {
+        do {
+            let defaultAccount = try WPAccount.lookupDefaultWordPressComAccount(in: context)
+
+            guard
+                let account = defaultAccount,
+                let restApi = account.wordPressComRestApi
+            else {
                 finished(false, nil)
                 return
-        }
+            }
 
-        let accountSettingService = AccountSettingsService(userID: defaultAccount.userID.intValue, api: restApi)
+            let accountSettingService = AccountSettingsService(userID: account.userID.intValue, api: restApi)
 
-        accountSettingService.updatePassword(newPassword) { (success, error) in
-            if success {
-                WordPressAuthenticator.track(.signupEpiloguePasswordUpdateSucceeded, properties: self.tracksProperties())
-            } else {
-                WordPressAuthenticator.track(.signupEpiloguePasswordUpdateFailed, properties: self.tracksProperties())
+            accountSettingService.updatePassword(newPassword) { (success, error) in
```

Comment:
Nitpick: The parenthesis should be unnecessary, right?

```suggestion
            accountSettingService.updatePassword(newPassword) { success, error in
```